File: backend/backendShop/backendShop/Models/Order.cs
using System;
using System.Collections.Generic;

namespace backendShop.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public DateTime TimeOrdered { get; set; }
        public bool IsCancelled { get; set; }
        public string Comment { get; set; }
        public string Address { get; set; }
        public float TotalCost { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }

        public virtual List<OrderProduct>? OrderItems { get; set; }
    }
}

File: frontend/webshop/src/models/Order.js
export default class Order {
  constructor(orderId, timeOrdered, comment, address, totalCost, userId, orderItems,isCancelled) {
    this.orderId = orderId;
    this.timeOrdered = timeOrdered;
    this.comment = comment;
    this.address = address;
    this.totalCost = totalCost;
    this.userId = userId;
    this.orderItems = orderItems || []; // Initialize with an empty array if not provided
    this.isCancelled = isCancelled;
  }
}